server/sonar-db-dao/src/main/resources/org/sonar/db/component/BranchMapper.xml

```diff
@@ -28,6 +30,7 @@
       #{dto.keyType, jdbcType=VARCHAR},
       #{dto.branchType, jdbcType=VARCHAR},
       #{dto.mergeBranchUuid, jdbcType=VARCHAR},
+      #{dto.pullRequestBlob, jdbcType=BLOB},
```

Comment:
I saw `BINARY` used instead of `BLOB` in most of the places in our code

Reply:
I followed the example in `FileSourceMapper`. It seems that's the only one using `BLOB` instead of `BINARY`. I changed it, along with other suggestions.

Reply:
this is one of the rare example where the column is declared as nullable explicitly. It might not change anything, but just in case. Have you found how to fix your problem?